grafana/provisioning/datasources/prometheus.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: 1
+
+datasources:
+- name: Prometheus
+  type: prometheus
+  access: server
+  orgId: 1
+  url: http://localhost:9090
+  version: 1
+  editable: true
+
```

Comment:
This YAML will provide our Prometheus datasource.

We will use this datasource to build our monitoring and load analysis dashboards, and potencially our Reimbursements analytics dashboards too.